cmd/influxd/restore/restore.go

```diff
@@ -166,6 +166,10 @@ func (cmd *Command) parseFlags(args []string) error {
 		return fmt.Errorf("backup path should be a valid directory: %s", cmd.backupFilesPath)
 	}
 
+	if cmd.destinationDatabase != "" && cmd.sourceDatabase == "" {
```

Comment:
Looking at this, a lot of the argument validation is in the `else` below, should we be enforcing some version of them regardless of portable/online?

Reply:
We want to do the check here for both branches of the if/else because they all end up in the same call to RequestMetaStoreUpdate RPC which will fail in the same way.